governance/votes/Executive vote - October 17, 2024.md

```diff
@@ -0,0 +1,287 @@
+  - [`cusp`](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.5.3_Max_Auction_Drawdown_(cusp)-083ada9d-8d45-46ae-80d2-a3bb706e1e77|57eaf45219bea3b430c2a59a): **0.40**.
+  - [`chip`](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.5.5_Proportional_Kick_Incentive_(chip)-afecb109-75a7-4289-997e-2a6f3d3bea1e|57eaf45219bea3b430c2a59a): **0.1%**.
+  - [`tip`](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.5.6_Flat_Kick_Incentive_(tip)-1c13687a-2dad-4e86-9940-024dc8f17a24|57eaf45219bea3b430c2a59a): **300 DAI**.
+  - `stopped`: **0**.
+  - [`tolerance`](https://sky-atlas.powerhouse.io/#A.1.8.2.2.4.1_Liquidations_Circuit_Breaker_Exception_Price_Tolerance-bcccd6b0-ad61-47c6-94f1-93447ba39102|0db30758e055352ce6eb): **0.5**.
+- StairstepExponentialDecrease calc contract will be configured with the following parameters:
+  - [`cut`](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.5.1.1_Auction_Price_Function_(cut)-b6e47d05-addf-4371-ad95-90582da7c0f1|57eaf45219bea3b430c2a59a6013): **0.99**.
+  - [`step`](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.5.1.2_Auction_Price_Function_(step)-7d58c961-43ff-4157-80aa-8a92f577ec12|57eaf45219bea3b430c2a59a6013): **60 seconds**.
+- REWARDS_LSMKR_USDS will be added to the Seal Engine as the first farm.
+- The LockstakeEngine contract will be authorized to access the `vat`.
+- The LockstakeClipper contract will be authorized to access the `vat` and the LockstakeEngine.
+- CLIPPER_MOM, MCD_DOG, and MCD_END will be authorized to access LockstakeClipper.
+- New [Chainlog](https://chainlog.sky.money) keys LOCKSTAKE_MKR, LOCKSTAKE_ENGINE, LOCKSTAKE_CLIP, and LOCKSTAKE_CLIP_CALC will be added.
+
+#### Early Bird Rewards Multisig Funding
+
+To facilitate the distribution of [Early Bird Rewards](https://sky-atlas.powerhouse.io/#A.5.2.1.1_Reward_System-99e76538-f850-4937-8092-d166cae5aff4|8d5a9e88cf49) **27,222,832.80 SKY** will be minted to a multisig controlled by the Accessibility Facilitators at [0x14D98650d46BF7679BBD05D4f615A1547C87Bf68](https://etherscan.io/address/0x14D98650d46BF7679BBD05D4f615A1547C87Bf68).
+
+Once the required SKY tokens have been transferred to the MerkleDistributor it is expected that the remaining tokens will be burned.
+
+### RWA Debt Ceiling Reductions
+
+- **Authorization**: [Forum post](https://forum.sky.money/t/2024-10-17-expected-executive-contents-rwa-vault-changes/25323/2)
+- **Proposal**: [Forum post](https://forum.sky.money/t/2024-10-17-expected-executive-contents-rwa-vault-changes/25323)
+
+If this executive proposal passes, then the following Debt Ceiling changes for the now deprecated RWA Vaults operated by Monetalis will be carried out.
+
+#### Monetalis Clydesdale (RWA007-A)
+
+- RWA007-A will be removed from the [Debt Ceiling Instant Access Module (DC-IAM)](https://sky-atlas.powerhouse.io#A.3.8.1.1.2.4_Debt_Ceiling_Instant_Access_Module_(DC_IAM)-071d42e3-8a21-4401-852e-0b52c49768bb|57eaf45219bea3b430c2).
+- RWA007-A Debt Ceiling will be set to **0 DAI**.
```

Comment:
```suggestion
- Reduce the RWA007-A Debt Ceiling by 3 billion DAI from 3 billion DAI to **0 DAI**.
```
This is more of a question than a suggestion. Why aren’t we using the same format as the Coinbase Custody Debt Ceiling change? Is it because Monetalis Clydesdale uses the DC-IAM?

Reply:
You are correct. Monetalis is on the DC-IAM while Coinbase Custody is not.

Reply:
And it has to be removed from the DC-IAM in order to set the DC to 0.

Reply:
Would you like to resolve this one @VoteWizard , or would you still like it changed? We feel it is correct as is, but don't mind the suggestion if you prefer it.